### DISCUSSION TITLE
[法律求助]我叫来家里的保洁打伤了三个入室抢劫犯

### DISCUSSION THREAD

**Discussion by Aris-Tendo** [share121/inter-knot#335] [其他]
![image](https://github.com/user-attachments/assets/c21b8701-11b6-45a5-8534-e06fa80ecd23)

今天我约了一个上门保洁服务。保洁人员因为堵车修路迟到了五分钟，一直跟我道歉。本来有点生气，但看到她态度那么卑微，有点不忍心，就算了。
我交代好保洁要求后就去看电影了。看到一半，治安局打电话来，说有三个人闯入我家抢劫。我想起家里还有保洁人员，还是个小姑娘，就问了下她还好吗，治安局说是那三个抢劫犯报的案——她把三个壮年男子打进了医院急诊…
现在三个人找律师向我索赔医药费，有懂法的朋友说说我该怎么办吗？

**Comment by en-461** [comment]
你这个保洁能不能介绍给我，我刚好需要一个女打手
